src/app/backend/replicasetdetail.go

```diff
@@ -135,6 +135,12 @@ func GetReplicaSetDetail(client *client.Client, namespace string, name string) (
 	return replicaSetDetail, nil
 }
 
+// Deletes replica set with given name in given namespace
+func DeleteReplicaSet(client *client.Client, namespace string, name string) error {
+	return client.ReplicationControllers(namespace).Delete(name)
```

Comment:
It looks like this is not enough. It only deletes Replication Controller without deleting even Pods. I have to take a look how `kubectl delete rc...` works.